jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp

```diff
@@ -38,6 +42,55 @@
  */
 
 /**
+ * Insert leading zeros to a string of a number if needed.
+ */
+static void
+ecma_date_insert_leading_zeros (ecma_string_t **str_p, /**< input/output string */
```

Comment:
I think `ecma_builtin_date_` will be too long for a prefix and we used `ecma_date_` for helpers in `ecma-builtin-date.cpp`